src/interfaces/chain.h

```diff
@@ -7,6 +7,7 @@
 
```

Comment:
In 4c64dd5a3cc4a48e5470fff26c99dde20f81c7e0   "Change MemPoolRemovalReason to variant type"
nit:  Commit message should follow the guidelines in  https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#committing-patches

The explanation is a long should break it.